tests/Manifest1.jad

```diff
@@ -3,6 +3,6 @@ MIDlet-Jar-Size: 2436
 MIDlet-Jar-URL: tests.jar
 MIDlet-Name: AMIDlet
 MIDlet-Vendor: MIDlet Suite Vendor
-MIDlet-Version: 1.0.0
+MIDlet-Version: 1

```

Comment:
That would be https://github.com/andreasgal/j2me.js/blob/f8e9a09acea4617bd22315eb3240cbb046828e43/libs/urlparams.js#L43.